README.md

```diff
@@ -250,27 +250,17 @@ export NVM_DIR="$HOME/.nvm"
 
 By design and just like `scripts` defined in `package.json`, husky will run hook scripts as a single command. 
 
-```json
+```jsonc
 "pre-commit": "cmd && cmd"
-```
 
-That said, if you prefer to use an array, the recommended approach is to define them in `.huskyrc.js` or `husky.config.js`.
+// or
 
-```js
-const tasks = arr => arr.join(' && ')
-
-module.exports = {
-  'hooks': {
-    'pre-commit': tasks([
-      'cmd',
-      'cmd'
-    ])
-  }
-}
+"pre-commit": [
+  "cmd",
+  "cmd"
+]
 ```
```

Comment:
Perhaps you can change the structure here slightly to make it clear in text that both options exists. Also, I think it is useful to specify the array option first. E.g.:

---

### Multiple commands

To run multiple commands in series you can specify an array of commands.

```json
"pre-commit": [
  "cmd1",
  "cmd2"
]
```

If you use a string instead, just like `scripts` defined in `package.json`, husky will always run hook scripts as a single command.

```json
"pre-commit": "cmd1 && cmd2"
```

:information_source: _The two examples above will result equivalent behavior._

---